config/package/tmpfiles.d/uwsgi.conf

```diff
@@ -0,0 +1 @@
+d /run/uwsgi/socorro 0755 socorro nginx -
```

Comment:
The advantage to this is that we can move the systemd service files into `./config/package/` too, they are in an odd place now.